.github/workflows/ci-cd.yml

```diff
@@ -41,9 +41,12 @@ jobs:
         os:
         - ubuntu-24.04
         - ubuntu-24.04-arm
-        experimental:
-        - false
+        # httpx=false for all runs, but then add a job on 3.13 with httpx
+        no-httpx: [false]
         include:
```

Comment:
All existing matrix dimensions need to be included in the new `include` entry. `experimental: false` is missing. Please also keep the order of fields consistent.

The existing `include` entries need to be modified also: Please add the new `no-httpx: false` field to each of them.